bin/all_sky_search/pycbc_coinc_findtrigs

```diff
@@ -163,6 +174,11 @@ if args.randomize_template_order:
 else:
     template_ids = range(tmin, tmax)
 
+# Only analyze templates which might have triggers based on
+# our 'tids_with_trigs'
```

Comment:
is it 'might have triggers' or 'do have triggers' ?  I guess what you want to say is 'might have coincs'

Also cut 'based on our tids_with_trigs' since it's obvious from code

Reply:
Done